### PR TITLE
Fixed a typo in MainFrame.cpp

### PR DESCRIPTION
**Pull request by justanotheranonymoususer**

### CODE REVIEW

File: DebugView++/MainFrame.cpp [hunk 1]
```diff
@@ -268,7 +268,7 @@ std::wstring FormatDuration(double seconds)
 	if (minutes > 0)
 		return wstringbuilder() << FormatUnits(minutes, L"minute") << L" " << FormatUnits(floor_to<int>(seconds), L"second");
 
-	static const wchar_t* units[] = { L"s", L"ms", L"�s", L"ns", nullptr };
+	static const wchar_t* units[] = { L"s", L"ms", L"µs", L"ns", nullptr };
```
Comment (justanotheranonymoususer) [inline DebugView++/MainFrame.cpp]: I edited the file online, and don't know why this is shown in the diff.

**Comment by janwilmans** [comment]
thanks!

File: DebugView++/MainFrame.cpp (hunk 1 above)
Comment (justanotheranonymoususer) [inline DebugView++/MainFrame.cpp]: Hmm, the encoding of µ changed to UTF-8. Perhaps this should be reverted. Sorry.

Comment (janwilmans) [inline DebugView++/MainFrame.cpp]: I seems to compile and work just fine, so no harm done.

On 12 March 2015 at 22:39, justanotheranonymoususer <
notifications@github.com> wrote:

> In DebugView++/MainFrame.cpp
> <https://github.com/djeedjay/DebugViewPP/pull/170#discussion_r26347085>:
>
> > @@ -268,7 +268,7 @@ std::wstring FormatDuration(double seconds)
> >  	if (minutes > 0)
> >  		return wstringbuilder() << FormatUnits(minutes, L"minute") << L" " << FormatUnits(floor_to<int>(seconds), L"second");
> >
> > -	static const wchar_t* units[] = { L"s", L"ms", L"�s", L"ns", nullptr };
> > +	static const wchar_t* units[] = { L"s", L"ms", L"µs", L"ns", nullptr };
>
> Hmm, the encoding of µ changed to UTF-8. Perhaps this should be reverted.
> Sorry.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/djeedjay/DebugViewPP/pull/170/files#r26347085>.
>



-- 
Met vriendelijke groeten,

Jan Wilmans

**Pull request merged**